test/transform/letTest.js

```diff
@@ -460,6 +460,15 @@ describe('Let/const', () => {
       );
     });
 
+    it('should ignore variable declared inside a block but modified outside', () => {
```

Comment:
It's not actually a modification. Variable `x` is not modified. Rather the point is that the variable is used in computed property access.